### PR TITLE
Port to 1.18.1

### PR DESCRIPTION
**Pull request by emilyastranova**
Could not get Forge to pull the new version from servers, don't merge this until that is fixed 🤣

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 modName=EditSign
-version=2.2.6
+version=2.2.7
 releaseType=release
 # Repository
 repoUser=RakSrinaNa
 repoName=EditSign
 # Forge
 mappings_channel=official
-mappings_version=1.18
```
Comment (Rakambda) [inline gradle.properties:15]: This should be just `1.18.1`. 
What you changed here are the mappings for forge. This is probably why you didn't managed to get Forge working.

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [versions]
 modmenu-version = "3.0.0"
-cloth-config-fabric-version = "6.0.43"
-cloth-config-forge-version = "6.0.43"
```
Comment (Rakambda) [inline gradle/libs.versions.toml:4]: This also has to be changed in the forge's build.gradle. `fg.deobf` doesn't support versions catalog so it is duplicated there.

**Review by Rakambda — CHANGES_REQUESTED** [review]

**Comment by Rakambda** [comment]
As an example this is [what it looks like](https://github.com/RakSrinaNa/FallingTree/pull/261) on one of my other mods (which is made the same way) 😉 

Also, can you target the 1.18.1 branch please.

**Comment by emilyastranova** [comment]
> As an example this is [what it looks like](https://github.com/RakSrinaNa/FallingTree/pull/261) on one of my other mods (which is made the same way) 😉 
> 
> Also, can you target the 1.18.1 branch please.

If I had the time I would, I worked to get my server ported over for my campus but now I'm floored with work. Feel free to scrap my code or use it for reference

**Review by Rakambda — APPROVED** [review]

**Pull request merged**